cli-feed-v4.json

```
@@ -24443,6 +24453,379 @@
+          "sha2": "4fbeac42a76c7265b64dd0967f398671ccfd739f7e906c40b22f87432f126d52",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.osx-x64.4.0.5780.zip",
+          "sha2": "fa9327e1b075f08a70d4b49e13837ed335fa98cf80e0ce26e9778154e92e5ac4",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "arm64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.osx-arm64.4.0.5780.zip",
+          "sha2": "77f6e29b957bc0d64ef76dea3325c686a16a4316fed6e616b61fc9e74e70690d",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-x64.4.0.5780.zip",
+          "sha2": "d139149f897a0a39a1c960e71e4bdd2a475e4fda4e1cba14c684c2b421aa1935",
+          "size": "minified",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.win-x86.4.0.5780.zip",
+          "sha2": "dc9917152c6becf8919e864933a9d137b52cdf0cc51ac5c1d6776e5fa0aae333",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-x86.4.0.5780.zip",
+          "sha2": "ed3234722696a8f621e3179a644955d2399591d61e25307486ab6ecc8165f7aa",
+          "size": "minified",
+          "default": "true"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "arm64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-arm64.4.0.5780.zip",
+          "sha2": "a31bd555992af47cebefa6e0e82afbe3831ab1df5cf8b525774fd9e17d4004aa",
+          "size": "minified",
+          "default": "false"
+        }
+      ]
+    },
+    "4.77.0-inprocess": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "net8": {
+            "displayInfo": {
+              "displayName": ".NET 8.0",
+              "hidden": false,
+              "displayVersion": "v4",
+              "targetFramework": ".NET",
+              "description": "LTS",
+              "endOfLifeDate": "2026-11-12T00:00:00Z"
+            },
+            "capabilities": "net8",
+            "sdk": {
+              "name": "Microsoft.NET.Sdk.Functions",
+              "version": "4.4.0"
+            },
+            "default": false,
```

I think this should indeed be default
```suggestion
            "default": true,
```